bridge/opencensus/internal/ocmetric/metric.go

```diff
@@ -152,22 +157,187 @@ func convertHistogram(labelKeys []ocmetricdata.LabelKey, ts []*ocmetricdata.Time
 				Sum:          dist.Sum,
 				Bounds:       dist.BucketOptions.Bounds,
 				BucketCounts: bucketCounts,
+				Exemplars:    exemplars,
 			})
 		}
 	}
 	return metricdata.Histogram[float64]{DataPoints: points, Temporality: metricdata.CumulativeTemporality}, err
 }
 
-// convertBucketCounts converts from OpenCensus bucket counts to slice of uint64.
-func convertBucketCounts(buckets []ocmetricdata.Bucket) ([]uint64, error) {
+// convertBuckets converts from OpenCensus bucket counts to slice of uint64,
+// and converts OpenCensus exemplars to OpenTelemetry exemplars.
+func convertBuckets(buckets []ocmetricdata.Bucket) ([]uint64, []metricdata.Exemplar[float64], error) {
 	bucketCounts := make([]uint64, len(buckets))
+	exemplars := []metricdata.Exemplar[float64]{}
```

Comment:
Should the Examplars be the same size as bucketCounts?

Reply:
The OTel [data model](https://github.com/open-telemetry/opentelemetry-proto/blob/ac3242b03157295e4ee9e616af53b81517b06559/opentelemetry/proto/metrics/v1/metrics.proto#L453) doesn't requires the number or order of exemplars to be the same as the set of histogram buckets counts, which must be ordered, and have a matching lenth.

Even it OTel, you would only get one exemplar per-bucket on histograms if you are using the AlignedHistogramBucketExemplarReservoir.  Otherwise, I believe you could end up with multiple exemplars per-bucket, or exemplars that are not aligned with buckets.

OpenCensus does limit each bucket to at most one exemplar, but I don't think we should insert empty exemplars for buckets that don't have one.